### PR TITLE
Set the correlation id and share project props

### PR DESCRIPTION
**Pull request by kceiw**
## Description
Use the shared variable to set the correlation id so that we can correlate the telemetry events in two tables.
Refactor the project files to share the common properties.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: tools/Az.Tools.Predictor/Az.Tools.Predictor/Telemetry/AzPredictorTelemetryClient.cs [hunk 1]
```diff
@@ -80,7 +78,7 @@ public virtual void OnHistory(HistoryTelemetryData telemetryData)
         {
             PostTelemetryData(telemetryData);
 
-            _commandId = Guid.NewGuid().ToString();
+            CommandId = Guid.NewGuid().ToString();
```
Comment (msftsiwei) [inline tools/Az.Tools.Predictor/Az.Tools.Predictor/Telemetry/AzPredictorTelemetryClient.cs:81]: if commandid already have a guid in property init place, then here we will force to refresh the guid again?

Comment (kceiw) [reply tools/Az.Tools.Predictor/Az.Tools.Predictor/Telemetry/AzPredictorTelemetryClient.cs:81]: Yes. Those Ids are updated once the cycle ends. CommandId is updated when a command is executed. RequestId is updated for a new request to the server etc.

**Review by mirdaki — APPROVED** [review]
Looks good to me

**Review by jjaguirre394 — APPROVED** [review]
Looks good.

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**